src/app.js

```diff
@@ -1 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+// fs.copyFile('./src/file.txt', './src/new_file.txt', error => {
+//   if (error) {
+//     throw error('There is not file in this directory');
+//   }
+//    console.log('File was coppied succesfull');
+// });
+
+const [prev, next] = process.argv.slice(2);
```

Comment:
It's bad names, I don't understand what exactly you do, what is it, prev and next?